postcss.config.js

```diff
@@ -0,0 +1,59 @@
+const { parse } = require('path')
+const autoprefixer = require('autoprefixer')
+const cssnano = require('cssnano')
+const oldie = require('oldie')
+const pseudoclasses = require('postcss-pseudo-classes')
+
+/**
+ * PostCSS config
+ *
+ * @param {object} context - PostCSS context
+ * @param {string} context.env - Browserslist environment
+ * @param {string | import('vinyl')} [context.file] - File path or object
+ */
+module.exports = ({ env, file = '' }) => {
+  const { dir, name } = parse(typeof file === 'object' ? file.path : file)
+
+  const plugins = [
+    autoprefixer({ env })
+  ]
+
+  // Minify CSS
+  if (name.endsWith('.min')) {
+    plugins.push(cssnano())
+  }
+
+  // Add review app auto-generated 'companion' classes for each pseudo-class
+  // For example ':hover' and ':focus' classes to simulate form label states
+  if (dir.endsWith('app/assets/scss') && (name === 'app' || name.startsWith('app-'))) {
+    plugins.push(pseudoclasses({
+      // Work around a bug in pseudo classes plugin that badly transforms
+      // :not(:whatever) pseudo selectors
+      blacklist: [
+        ':not(',
+        ':disabled)',
+        ':first-child)',
+        ':last-child)',
+        ':focus)',
+        ':active)',
+        ':hover)'
+      ]
+    }))
+  }
+
+  // Transpile CSS for Internet Explorer
+  if (name.endsWith('-ie8')) {
```

Comment:
I don't think this is doing what we want when running `npm run build:dist` because at that point we're generating a minified CSS file which ends with `-ie8.min`, so this plugin doesn't get added.

Reply:
Well spotted, this was a well-intentioned change from yesterday (previously `.includes()`) so I'll put it back!

Reply:
Done. For clarity I've matched on both:

```mjs
if (name.endsWith('-ie8') || name.endsWith('-ie8.min')) {
```

Plus I've added tests for each combination we rely on